app/src/components/main/analytics/events/ga4Events/projectSettingsPageEvents.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const PROJECT_SETTINGS = 'project_settings';
+export const PROJECT_SETTINGS_PAGE_EVENTS = {
```

Comment:
Maybe better return old name `PROJECT_SETTINGS_ANALYZER_EVENTS` inside file `projectSettingsPageEvents.js
there are two ways: create independent files for each subpage and put them into one folder which represent page or store all objects inside on file. 
As we have not so much events in settings we can store them in one file but split by menu items